libs/scopechangerequestsidesheet/src/lib/ui-sidesheet/tabs/LogTab.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  DateCell,
+  DescriptionCell,
+  StyledTabContent,
+  TabTable,
+} from '@cc-components/shared';
+import { StyledContentWrapper } from '@cc-components/sharedcomponents';
+import { Checkbox } from '@equinor/eds-core-react';
+import { LogEntry, ScopeChangeRequest } from '@cc-components/scopechangerequestshared';
+import { WorkflowChart } from '../WorkflowChart';
+import { VerticalLine } from './requestTab.styles';
+
+type LogTabProps = {
+  logEntry?: LogEntry[] | undefined;
+  isFetching?: boolean;
+  error?: Error | null;
+};
+
+export const Logtab = ({ logEntry, error, isFetching }: LogTabProps): JSX.Element => {
+  return (
```

Comment:
There should be some if(error), if(isFetching) in this file. This can be added in the next pr